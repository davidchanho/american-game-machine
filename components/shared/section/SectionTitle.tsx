import React from "react";
import AppearText from "../appear-text";

interface SectionTitleProps {
  title: string;
}

function SectionTitle({ title }: SectionTitleProps) {
  return (
    <AppearText>
      <p className="anchor mb-5 text-capitalize">
        <span id={title} className="d-block position-relative hidden" />
        <h3 className="text-center">{title}</h3>
      </p>
    </AppearText>
  );
}

export default SectionTitle;
