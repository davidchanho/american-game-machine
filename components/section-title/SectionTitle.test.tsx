import React from "react";
import renderer from "react-test-renderer";
import SectionTitle from ".";

describe("SectionTitle", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<SectionTitle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
