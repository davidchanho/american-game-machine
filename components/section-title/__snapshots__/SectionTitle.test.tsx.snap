// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle renders correctly 1`] = `
<h3
  className="sc-bdfBwQ gDPGBl"
/>
`;
